packages/components/buttongroup/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@contentful/f36-buttongroup",
```

Comment:
I'd probably make it a part of `button` package for the simplicity

Reply:
Should we add the `ToggleButton` and other types also on the button package?

Reply:
others are debatable, but `ButtonGroup` can't be used without a `Button` so for me it makes sense to colocate them within one package.

Reply:
I think `ToggleButton` would also be the case, as it will use the Button as base and only compose on top of it, and would also be usable on the `ButtonGroup`, but we can check the others later

Reply:
if so, we can probably collocate it too